go/test/endtoend/vreplication/resharding_workflows_v2_test.go

```diff
@@ -244,6 +246,73 @@ func TestBasicV2Workflows(t *testing.T) {
 	log.Flush()
 }
 
+/*
+testVSchemaForSequenceAfterMoveTables checks that the related sequence tag is migrated correctly in the vschema
+while moving a table with an auto-increment from sharded to unsharded.
+*/
+func testVSchemaForSequenceAfterMoveTables(t *testing.T) {
+	// at this point the unsharded product and sharded customer keyspaces are created by previous tests
+
+	// use MoveTables to move customer2 from product to customer using
+	currentWorkflowType = wrangler.MoveTablesWorkflow
+	err := tstWorkflowExec(t, defaultCellName, "wf2", sourceKs, targetKs,
+		"customer2", workflowActionCreate, "", "", "")
+	require.NoError(t, err)
+	time.Sleep(1 * time.Second)
```

Comment:
I wonder if this will introduce some flakiness? In my experience the new Workflow Create call will return an error because it does not start within the default wait timeout in some percentage of cases. In these 2 lines, we just want to confirm that it has successfully started, right? If so, can't we check the Show output instead to reduce flakiness?